README.md

```diff
@@ -14,7 +14,7 @@ Links:
 
 ## Latest Release
 
-Winter '22 ☃️ ☕ 🎿
+Summer '22 :sun_with_face: :sunglasses: :beach_umbrella:
```

Comment:
To create a PR for `feature/238`, we had to have at least one diff.  If there are no other diffs, we won't commit this change + release for 238.